src/pages/events/troubleshooting.md

```diff
@@ -26,11 +26,15 @@ You can use the following SQL query to select all events from the `event_data` t
 
 `SELECT * FROM event_data`
 
-Use the query results to determine the next troubleshooting step.
+Alternatively, the _Event Status_ grid, found by navigating to **System** > Events > **Event Status** in the Commerce admin, provides a way to view the status of triggered events contained in the `event_data` table through the UI.
```

Comment:
```suggestion
You can also use the _Event Status_ grid in the Commerce Admin to display the status of triggered events that are present in the `event_data` table. Select **System** > Events > **Event Status** to view this grid.
```
I left this as a suggestion rather than editing it myself because I think the UI solution should be mentioned before encouraging the reader to go into the DB. Is there any reason to mention the DB at all? Are there edge cases that can only be resolved by digging into the DB?

Reply:
The DB table contains the payload for each event, but we decided against showing the payload for events in the UI. I think there could be edge cases where a user may be trying to figure out why a specific event payload has not yet been received in app builder. For example, if they're looking for an `observer.customer_login` event  with a specific payload that they were expecting in app builder but there have been multiple `observer.customer_login` events triggered recently, the UI wouldn't be very helpful in figuring out which of those events is the one with the payload of interest